### DISCUSSION TITLE
İngilizce dersindeki mantık sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#252] [Beyin Fırtınası]
Öğretmen ingilizce dersinde tahtaya alt alta şu kelimeleri yazmış:
cat
dog
has
max
dim
tag
Sonra da kimseye söylemeden bu kelimelerin birini seçmiş ve seçtiği kelimenin harflerini Ayşe, Betül ve Canan’a dağıtmış. Her öğrenci sadece bir harf almış. Sonra bu üçü kendilerine verilen harflere bakmış ama sadece kendi harflerine. Ardından öğretmen sormaya başlamış.
Öğretmen: Ayşe, senin harfinin hangi kelimeden geldiğini bulabildin mi?
Ayşe: Evet, buldum.
Öğretmen: Betül, peki sen harfinin hangi kelimeden olduğunu bulabildin mi?
Betül: Hmmm, evet buldum şimdi.
Öğretmen: Canan, sen de bulabildin mi?
Canan: Biraz düşüneyim... Tamam, buldum.
Hangi öğrenciye hangi harf verilmişti?

**Comment by Murkrow067** [comment]
Ayşe kendi kelimesini direkt bulabildiğine göre elindeki harf yalnızca bir kelimenin içerisinde bulunuyor olmalı. Buna uyan seçenekler: i, x, s, h, o ve c. Betül ve Canan ise kendilerinden önce gelen cevap sayesinde kendi kelimelerini bulabildikleri için ellerindeki harf tahtadaki kelimlerin içerisinde birden fazla kez bulunuyor olmalı. Bu seferki seçeneklerimiz: t, a, g, m ve d. Bu verilere uyan kelimelerimiz: cat, dog, max ve dim. Elenen kelimelerden sonra kendi kelimesini bulabildiğine göre Betül'ün elindeki harf kalan kelimelerin içerisinde yalnızca birer defa bulunan t veya g olmalı. Bunlara dayanarak diyebiliriz ki: Ayşe'ye o/c, Betül'e g/t ve Canan'a d/a harfleri verilmiştir.
